packages/expo-dev-launcher/bundle/views/LauncherApp.tsx

```diff
@@ -31,10 +34,13 @@ function LauncherApp(props) {
     };
   });
 
+  const backgroundColor = theme === 'dark' ? Colors.dark.background : Colors.light.background;
+  const statusBarContent = theme === 'dark' ? 'light-content' : 'dark-content';
```

Comment:
```suggestion
  const { colors, dark } = useTheme();
  const backgroundColor = colors.background;
  const statusBarContent = dark ? 'light-content' : 'dark-content';
```
Perhaps? This way all the colors are defined in the theme?